### PR TITLE
Feature/openrgb fix

### PR DESCRIPTION
**Pull request by MatrixDJ96**
**This pull request proposes the following changes:**
- Improve OpenRGB devices

### CODE REVIEW

File: Project-Aurora/Project-Aurora/Devices/OpenRGB/OpenRGBDevice.cs [hunk 1]
```diff
@@ -33,291 +33,279 @@ public override bool Initialize()
...
+                // Search if process is running
+                while (!openrgbRunning())
                 {
                     Thread.Sleep(100);
                     remainingMillis -= 100;
                     if (remainingMillis <= 0)
                     {
-                        _openRgb = null;
-                        return false;
+                        throw new Exception("OpenRGB is not running...");
```
Comment (Aytackydln) [inline Project-Aurora/Project-Aurora/Devices/OpenRGB/OpenRGBDevice.cs:50]: can you also update DeviceDetails property? This was the message will be shown on UI, too.

File: Project-Aurora/Project-Aurora/Devices/OpenRGB/OpenRGBDevice.cs [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
     {
         public override string DeviceName => "OpenRGB";
-        protected override string DeviceInfo => string.Join(", ", _devices.Select(d => d.OrgbDevice.Name));
+        protected override string DeviceInfo => string.Join(", ", _helpers.Select(h => h.Device.Name));
 
-        private OpenRgbClient _openRgb;
-        private List<HelperOpenRgbDevice> _devices;
+        public readonly Queue<DeviceKeys> MouseLights = new Queue<DeviceKeys>(OpenRgbKeyNames.MouseLights);
 
-        private readonly object _updateLock = new();
```
Comment (Aytackydln) [inline Project-Aurora/Project-Aurora/Devices/OpenRGB/OpenRGBDevice.cs:27]: is this okay? Race conditions might break some stuff while device list is updating. Can you do "Rescan devices" on OpenRGB while Aurora is running a few times?

**Comment by Aytackydln** [comment]
Thank you for trying to help Aurora but I will be removing those files in favor of RGB.NET implementation soon.
So I suggest no longer continuing your work on this but instead you can improve rgb.net implementation

File: Project-Aurora/Project-Aurora/Devices/OpenRGB/OpenRGBDevice.cs (hunk 1 above)
Comment (MatrixDJ96) [reply Project-Aurora/Project-Aurora/Devices/OpenRGB/OpenRGBDevice.cs:50]: Yes!

**Comment by MatrixDJ96** [comment]
> 

Oh, ok! Do you need help to develop with new feature?

**Pull request closed without merging**